bot/configuration.py

```diff
@@ -422,6 +431,15 @@ def __init__(self, config: configparser.ConfigParser):
             + self.status_message_content.parsing_errors
         )
 
+    def dump_config_to_log(self):
+        with open(self.bot_config.log_file, "a", encoding="utf-8") as log_file:
+            log_file.write("\n*******************************************************************\n")
+            log_file.write("Current Moonraker telegram bot config\n")
```

Comment:
Since the includes have been read in to the config at this point, would it make sense to strip "include" sections from the dump?

Reply:
good offer. Added cleaning.